src/components/HikeComposer/ExportModal.js

```diff
@@ -35,30 +39,30 @@ const ExportModal = ({ visible, jsonData, onClose, branchName, category }) => {
     }
   };
 
-  const visitFork = () => {
-    window.open(
-      `https://github.com/HCL-TECH-SOFTWARE/volt-mx-tutorials/compare/${BASE_BRANCH}...mikeangelsilva:${branchName}?expand=1`,
-      "_blank"
-    );
-  };
+  // const visitFork = () => {
```

Comment:
Please remove the commented code. If it is required for re-enablement in the future please comment and add a TODO statement.

Reply:
removed